assets/src/containers/ResourcesAccessed.js

```diff
@@ -236,7 +237,7 @@ function ResourcesAccessed (props) {
       <Grid container spacing={2}>
         <Grid item xs={12}>
           <Paper className={classes.paper}>
-            <Typography variant='h5' gutterBottom className='title'>Resources Accessed</Typography>
```

Comment:
Since you removed the Typography can you also remove the import as well?

Reply:
Oh, yeah, I'll check to see if we can remove that, thanks.

Reply:
This should be resolved now.